src/index.css

```diff
@@ -1,12 +1,13 @@
+@import url("https://fonts.googleapis.com/css2?family=Aboreto&family=Cinzel&family=DM+Sans:opsz,wght@9..40,700&display=swap");
 
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif, "DM Sans",
+    sans-serif;
```

Comment:
we need the first font to be ours, the DM Sans, then we use the system-ui and whatever other fonts you want as a fallback. Right now your used font is ui-sans-serif on the page